src/GDGUkraine/rest_controller.py

```diff
@@ -372,77 +373,65 @@ def export_participants(self, id):
-                (lambda x: x.User.experience_desc or ''),
-                (lambda x: x.User.interests or ''),
-                (lambda x: x.User.events_visited or ''),
-                (lambda x: x.User.english_knowledge or ''),
-                (lambda x: x.User.t_shirt_size or ''),
-                (lambda x: x.User.additional_info or ''),
-                (lambda x: (
-                    json.dumps(x.EventParticipant.fields)
-                    if x.EventParticipant.fields
-                    else ''
-                )),
-                (lambda x: x.EventParticipant.confirmed),
-            ],
-            headers=[
-                'Registration id',
-                'Registration date',
-                'Event id',
-                'Event title',
-                'Name',
-                'Gender',
-                'Nickname',
-                'Email',
-                'Phone',
-                'Google +',
-                'City',
-                'Company',
-                'Position',
-                'Website',
-                'Experience level',
-                'Experience description',
-                'Interests',
-                'Events visited',
-                'English knowledge',
-                'T-Shirt size',
-                'Additional info',
-                'Extra fields',
-                'Confirmed',
-            ],
-        )
+
+        # Set appropriate headers
+        filename = re.compile(r'[^\w-]').sub('', event.title.replace(' ', '_'))
+
         cherrypy.response.headers['Content-Type'] = (
             'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
         )
-        filename = re.compile(r'[^\w-]').sub('', event.title.replace(' ', '_'))
         cherrypy.response.headers['Content-Disposition'] = (
             'attachment; filename={}-{}-{}-participants.xlsx'.format(
                 event.id, filename, event.date,
             )
         )
-        return file_generator(exporter.get_xlsx_content())
+
+        # Retrieve participation data
+        part_data = api.find_participants_by_event(orm_session, event)
+
+        return file_generator(gen_participants_xlsx(part_data))
+
+    @cherrypy.tools.json_out()
+    @cherrypy.tools.authorize()
+    def generate_invites(self, id):
+        return {}
+
+    @cherrypy.tools.json_out()
+    @cherrypy.tools.authorize()
+    def generate_report(self, id, mode=None):
+        """Exports spreadsheet file with event participants to Google Drive
+
+        Args:
+            id (int): event id
+        """
+        id = int(id)
+        req = cherrypy.request
+        orm_session = req.orm_session
+
+        # Retrieve event object
+        event = api.find_event_by_id(orm_session, id)
+        if event is None:
+            raise HTTPError(404)
+
+        file_mime = ('application/vnd.openxmlformats-officedocument'
+                     '.spreadsheetml.sheet')
+        file_name = 'Participants of [#{}] {} on {}'.format(
```

Comment:
Nope, but why shouldn't it work well? Python 3 is unicode-aware and Google Drive supports unicode names. I hope it uses UTF by default to decode incoming JSON metadata.